scripts/lib/canary.sh

```diff
@@ -4,8 +4,15 @@
 
 SECONDS=0
 
+# Fallback to KUBE_CONFIG_PATH if KUBECONFIG is not set
+
+if [ -n "$KUBE_CONFIG_PATH" ]; then
```

Comment:
This looks good. let's slight modify it like this.

```
# Fallback to KUBE_CONFIG_PATH,if set, and and KUBECONFIG is not defined.
if [ -n "$KUBE_CONFIG_PATH" ] && [ -z "$KUBECONFIG" ]; then
  export KUBECONFIG=$KUBE_CONFIG_PATH
fi
```

* This way, we can preserve the backwards compatible if some script is using older variable.
* And safely remove the dependency on `KUBE_CONFIG_PATH` by using `KUBECONFIG`

Reply:
Please replace this in other places of this usage too, and this looks good to me.

Reply:
Kindly check my latest commit